src/gui/interface/emotes.py

```diff
@@ -186,19 +187,19 @@ class NPCEmoteManager(EmoteManager):
     def __init__(
             self,
             emotes: dict[str, list[pygame.Surface]],
-            groups: tuple[pygame.sprite.Group, ...] | pygame.sprite.Group
+            *groups: pygame.sprite.Group
     ):
         """
         EmoteManager for all NPCs
         """
-        super().__init__(emotes, groups)
+        super().__init__(emotes, *groups)
 
 
 class EmoteWheel(EmoteWheelBase):
     def __init__(
             self,
             emote_manager: EmoteManagerBase,
-            groups: tuple[pygame.sprite.Group, ...] | pygame.sprite.Group,
```

Comment:
Actually it is, because in [ll. 235-236](https://github.com/sloukit/pydew-valley-uzh/pull/98/files/a714a59a3080dea8a525057dd58305fdaf9fbfba#diff-a6882cd9884a3e45d532d2f0fb102c70dbacd7241852cd8014d7f65dd877f68cR235-R236) the EmoteWheel has to be added as persistent Sprite to avoid having to add it to AllSprites every time a new map is loaded